test/podFailover/pod-failover-workload-chart/1pod1pvc.yaml

```diff
@@ -0,0 +1,23 @@
+storageClass:
+  name: azdisk-pod-failover-1pod1pvc-sc
+  parameters:
+    cachingmode: None
+    skuName: Premium_LRS
+    maxShares: "2"
```

Comment:
Does this mean we have replicas in this test? We also need to run test with no replicas and v2 driver.

Reply:
Here are all the driver/storage types:
1. Stateless
2. azdisk v1
3. azdisk v2
4. azstor
5. files

test types:
1. 1pod 1 pvc
2. 1 pod 3 pvc
3. 3 pod 3 pvc
For stateless you run 1 and 3.

failure types:
1. delete pod with node drain
2. delete pod without node drain

Reply:
Changes look good otherwise, just make sure we have all the coverage.